modules/silicon/README.md

```diff
@@ -0,0 +1,130 @@
+# RAD Lab Silicon Module
+
+## GCP Products/Services 
+
+* AI Platform Notebooks
+* BigQuery
+* Cloud Storage
+* Virtual Private Cloud (VPC)
```

Comment:
is the VPC stuff strictly required for all radlab modules? or is it possible to scrap it for simplicity?

Reply:
It's not strictly required, but it's best practice to create one.  Customers rarely use the default VPC and it's a recommendation to delete it anyway.  Most customers will have an org policy enabled that automatically deletes it, so we can't really make the assumption that it exists.

Reply:
Ack.